driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -185,15 +194,17 @@ public void apply(final ServerSettings.Builder builder) {
 
         helper = new JsonPoweredCrudTestHelper(description, database, database.getCollection(collectionName, BsonDocument.class));
 
-        ClientSession sessionZero = createSession("session0");
-        ClientSession sessionOne = createSession("session1");
+        if (serverVersionAtLeast(3, 6) && !isStandalone()) {
```

Comment:
The new JSON tests can be run on server versions prior to 3.6. We do not want to create sessions if sessions are unsupported on the server.